backend/config/config.go

```diff
@@ -141,13 +157,18 @@ type Syncing struct {
 	WarmupDuration time.Duration
 	Interval       time.Duration
 	TimeoutPerPeer time.Duration
+	// InboundDisable disables syncing content to the remote peer from our peer.
+	// If false, then documents get synced in both directions.
+	InboundDisable bool
 }
 
 // P2P configuration. For field descriptions see SetupFlags().
 type P2P struct {
 	Port           int
 	NoRelay        bool
+	NoDocumentsOut bool
 	BootstrapPeers []multiaddr.Multiaddr
+	DisableListing bool
```

Comment:
Consider renaming this to `NoListing` to be coherent with the rest of the disabling flags.